pipelines/skylab/smartseq2_multisample/MultiSampleSmartSeq2.wdl

```diff
@@ -0,0 +1,113 @@
+version 1.0
+
+import "../smartseq2_single_sample/SmartSeq2SingleSample.wdl" as single_cell_run
+import "../../../tasks/skylab/LoomUtils.wdl" as LoomUtils
+       
+workflow MultiSampleSmartSeq2 {
+  meta {
+    description: "The MultiSampleSmartSeq2 pipeline runs multiple SS2 samples in a single pipeline invocation"
+  }
+
+  input {
+      # Gene Annotation
+      File genome_ref_fasta
+      File rrna_intervals
+      File gene_ref_flat
+
+      # Reference index information
+      File hisat2_ref_name
+      File hisat2_ref_trans_name
+      File hisat2_ref_index
+      File hisat2_ref_trans_index
+      File rsem_ref_index
+
+      # Sample information
+      String stranded
+      String file_prefix
+      Array[String] input_file_names
+      String batch_id
+      Boolean paired_end
+  }
+  # Version of this pipeline
+  String pipeline_version = "MultiSampleSmartSeq2_v2.0.1"
```

Comment:
Change this to mimic dsde-pipelines where the name of pipeline isn't included in the version variable